components/mpas-seaice/src/Registry.xml

```diff
@@ -3737,6 +3785,52 @@
 		/>
 	</var_struct>
 
+	<!--1D atmos forcing -->
+	<var_struct name="1D_atmos_forcing" time_levs="1">
+		<var name="shortwaveDown1D"
+		     type="real"
+		     dimensions="Time"
+		     units="W m-2"
+		     description="Incoming shortwave flux (positive down)"
+		/>
+		<var name="longwaveDown1D"
+		     type="real"
+		     dimensions="Time"
+		     units="W m-2"
+		     description="Incoming longwave flux (positive down)"
+		/>
+		<var name="airTemperature1D"
```

Comment:
I think this implementation could be greatly simplified if the input forcing files included a nCells=1 dimension so that an airTemperature1D variable etc in addition to the airTemperature variable would not be needed

Reply:
@akturner : Agreed.  I chose to make the forcing data grid independent though so I could test on an arbitrary grid.  It runs with the QU240, for example, at least in short tests.

Reply:
@njeffery If the forcing is the same everywhere on the mesh, what's the utility of being able to run on meshes other than the single cell one?

Reply:
@akturner : your point is well taken.  I'll restructure the test case with a cell index in the forcing files. I'd like to keep the bgc nutrient file grid independent so that I can use it more generally, i.e. in a standard GCASE but with ocean bgc off.  Does that seem reasonable?